sdk/core/azure-core/src/main/java/com/azure/core/util/polling/LongRunningOperationStatus.java

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.util.polling;
+
+import com.azure.core.util.ExpandableStringEnum;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * An enum to represent all possible states that a long-running operation may find itself in.
+ * The poll operation is considered complete when the status is one of {@code SUCCESSFULLY_COMPLETED},
+ * {@code USER_CANCELLED} or {@code FAILED}.
+ */
+public final class LongRunningOperationStatus extends ExpandableStringEnum<LongRunningOperationStatus> {
```

Comment:
If the intention is to not use a generic name like OperationStatus,
Then PollOperationStatus is a good candidate